q2_cutadapt/plugin_setup.py

```diff
@@ -246,6 +247,9 @@
                       'allowable error rate. The default value specified by '
                       'cutadapt is 0.1 (=10%), which is greater than '
                       '`demux emp-*`, which is 0.0 (=0%).',
+        'batch_size': 'The number of samples to demultiplex at a time. Useful '
+                      'for working with datasets that consist of many samples '
+                      '. Set to "0" to process all samples at once.'
```

Comment:
```suggestion
        'batch_size': 'The number of samples cutadapt demultiplexes '
                      'concurrently. Demultiplexing in smaller batches will '
                      'yield the same result with marginal speed loss, and may'
                      ' solve \'too many files\' errors from many-sample data '
                      'sets. Set to "0" to process all samples at once.'
```
How's this? I'm trying to prevent the following misinterpretations:
- user thinks they need to demux multiple times with this param set
- user with not-too-many samples proactively runs this because "useful"

Reply:
I love it! How about one minor thought though: `errors from many-sample data sets.` -> `errors related to sample quantity.` (or something along those lines).

Reply:
:+1: